apps/admin-gui/src/app/shared/components/dialogs/create-attribute-definition-dialog/create-attribute-definition-dialog.component.ts

```diff
@@ -84,11 +88,30 @@ export class CreateAttributeDefinitionDialogComponent {
     this.attributeService
       .createAttributeDefinition({ attribute: this.attDef.getValue() })
       .pipe(
-        switchMap((attDef) => zip(of(attDef.id), of(this.collections))),
+        switchMap((attDef) => {
+          this.attDefCreated = attDef;
+          return zip(of(attDef.id), of(this.collections));
+        }),
         this.attributeRightsService.addAttributeId(),
         this.attributeRightsService.filterNullInPolicy(),
         switchMap((collections) =>
           this.attributeService.setAttributePolicyCollections({ policyCollections: collections })
+        ),
+        switchMap(() =>
+          this.attributeRightsService.updateAttributeAction(
+            this.finalReadOperations,
```

Comment:
I realized that if none of this calls depend on each other or have to be in order, we could use `mergeMap` to start all of them at once. If order is important then `concatMap` is probably better solution than chaining switch maps.